Tests/CacheAdvanceTests/CacheAdvanceTests.swift

```diff
@@ -602,7 +602,7 @@ final class CacheAdvanceTests: XCTestCase {
         throws
         -> CacheAdvance<TestableMessage>
     {
-        return try CacheAdvance<TestableMessage>(
```

Comment:
saw this while I was here. This change is very much unrelated to the goal of this PR